### PR TITLE
Update to the latest Dart Analysis Server protocol: 1.32.7

### PR DESCRIPTION
**Pull request by jwren**
This change adds the constructor tear-off highlighting support from the DAS, and removes some deprecated types, such as DartFix, from the DAS protocol.

https://github.com/dart-lang/sdk/issues/47160

### CODE REVIEW

File: Dart/resources/messages/DartBundle.properties [hunk 1]
```diff
@@ -22,6 +22,7 @@ dart.color.settings.description.symbol.literal=Symbol literal
 dart.color.settings.description.annotation=Annotation
 dart.color.settings.description.class=Class
 dart.color.settings.description.constructor=Constructor
+dart.color.settings.description.constructor.tearoff=Constructor tear off
```
Comment (alexander-doroshko) [inline Dart/resources/messages/DartBundle.properties:25]: Looking through the [spec](https://github.com/dart-lang/language/blob/master/accepted/future-releases/constructor-tearoffs/feature-specification.md) I noticed that noun is usually spelled as `tear-off`, and verb as `tear off`. Should it be `tear-off` in color settings?

File: Dart/src/com/jetbrains/lang/dart/highlight/DartSyntaxHighlighterColors.java [hunk 2]
```diff
@@ -54,11 +57,13 @@
   public static final String DART_STATIC_GETTER_REFERENCE = "DART_STATIC_GETTER_REFERENCE";
   public static final String DART_STATIC_METHOD_DECLARATION = "DART_STATIC_METHOD_DECLARATION";
   public static final String DART_STATIC_METHOD_REFERENCE = "DART_STATIC_METHOD_REFERENCE";
+  public static final String DART_STATIC_METHOD_TEAR_OFF = "DART_STATIC_METHOD_TEAR_OFF";
```
Comment (alexander-doroshko) [inline Dart/src/com/jetbrains/lang/dart/highlight/DartSyntaxHighlighterColors.java:60]: Should it also be registered in `DartAnnotator`?

File: Dart/src/com/jetbrains/lang/dart/highlight/DartSyntaxHighlighterColors.java [hunk 3]
```diff
@@ -54,11 +57,13 @@
...
   public static final String DART_STATIC_SETTER_DECLARATION = "DART_STATIC_SETTER_DECLARATION";
   public static final String DART_STATIC_SETTER_REFERENCE = "DART_STATIC_SETTER_REFERENCE";
 
   public static final String DART_TOP_LEVEL_FUNCTION_DECLARATION = "DART_TOP_LEVEL_FUNCTION_DECLARATION";
   public static final String DART_TOP_LEVEL_FUNCTION_REFERENCE = "DART_TOP_LEVEL_FUNCTION_REFERENCE";
+  public static final String DART_TOP_LEVEL_FUNCTION_TEAR_OFF = "DART_TOP_LEVEL_FUNCTION_TEAR_OFF";
```
Comment (alexander-doroshko) [inline Dart/src/com/jetbrains/lang/dart/highlight/DartSyntaxHighlighterColors.java:66]: Should it also be registered in `DartAnnotator`?

File: Dart/src/com/jetbrains/lang/dart/highlight/DartColorsAndFontsPage.java [hunk 4]
```diff
@@ -207,13 +215,16 @@
...
       new AttributesDescriptor(DartBundle.message("dart.color.settings.description.static.method.reference"), STATIC_METHOD_REFERENCE),
+      new AttributesDescriptor(DartBundle.message("dart.color.settings.description.static.method.tearoff"), STATIC_METHOD_TEAR_OFF),
       new AttributesDescriptor(DartBundle.message("dart.color.settings.description.static.setter.declaration"), STATIC_SETTER_DECLARATION),
       new AttributesDescriptor(DartBundle.message("dart.color.settings.description.static.setter.reference"), STATIC_SETTER_REFERENCE),
 
       new AttributesDescriptor(DartBundle.message("dart.color.settings.description.top.level.function.declaration"),
                                TOP_LEVEL_FUNCTION_DECLARATION),
       new AttributesDescriptor(DartBundle.message("dart.color.settings.description.top.level.function.reference"),
                                TOP_LEVEL_FUNCTION_REFERENCE),
+      new AttributesDescriptor(DartBundle.message("dart.color.settings.description.top.level.function.reference"),
```
Comment (alexander-doroshko) [inline Dart/src/com/jetbrains/lang/dart/highlight/DartColorsAndFontsPage.java:226]: Thanks to the 'Unused property' inspection, I noticed a copy/paste error here :)

**Review by alexander-doroshko — CHANGES_REQUESTED** [review]

File: Dart/resources/messages/DartBundle.properties (hunk 1 above)
Comment (jwren) [reply Dart/resources/messages/DartBundle.properties:25]: I went back and forth, but don't have a strong preference, changing to `tear-off`

File: Dart/src/com/jetbrains/lang/dart/highlight/DartSyntaxHighlighterColors.java (hunk 3 above)
Comment (jwren) [reply Dart/src/com/jetbrains/lang/dart/highlight/DartSyntaxHighlighterColors.java:66]: Added

File: Dart/src/com/jetbrains/lang/dart/highlight/DartSyntaxHighlighterColors.java (hunk 2 above)
Comment (jwren) [reply Dart/src/com/jetbrains/lang/dart/highlight/DartSyntaxHighlighterColors.java:60]: Added

**Pull request closed without merging**